pkg/regressionallowances/types.go

```diff
@@ -95,6 +96,9 @@ func addIntentionalRegression(release release, in IntentionalRegression) error {
 	if len(in.ReasonToAllowInsteadOfFix) == 0 {
 		return fmt.Errorf("reasonToAllowInsteadOfFix must be specified")
 	}
+	if len(in.JiraBug) == 0 {
+		return fmt.Errorf("jiraBug must be specified")
+	}
```

Comment:
If we always expect a link WDYT about?

```suggestion
if _, err := url.ParseRequestURI(in.JiraBug); err != nil {
	return fmt.Errorf("jiraBug must be a valid URL")
}
```

Reply:
good idea.  Changed.